resources/assets/lib/chat/conversation-view.tsx

```diff
@@ -94,6 +96,15 @@ export default class ConversationView extends React.Component<Props> {
 
     makeObservable(this);
 
+    disposeOnUnmount(
+      this,
+      observe(core.windowFocusObserver, 'hasFocus', (change) => {
```

Comment:
just a minor thing but according to the docs [it's preferred to use reaction over observe](https://mobx.js.org/intercept-and-observe.html#intercept--observe-) due to how observe works (ignores action and just run right away)